.gitignore

```diff
@@ -64,6 +64,6 @@ typings/
 zos.dev-*
 
 # contracts build
-client/src/contracts
```

Comment:
Can you double-check (with a clean checkout) if the react app still works after this change? I understand that create-react-app cannot retrieve contracts from outside the src folder, hence this configuration. Perhaps we could add a symlink?

Reply:
I had to fix this issue in order to make hot loader happen, because it pulls `.sol` files from outside directory. And react doesn't depend on `.json` files anymore. It only depends on `.sol` files thanks to solidity-hot-loader.

It is disabled at `config-overrides.js`.

I'll fix remaining issues in zepkit repo and then will test zepkit on clean checkout. Hope this is fine.

Reply:
Sure! Just post here whenever you have tested with a clean checkout, and if it's fine, we'll merge.